src/backend/gporca/libgpopt/src/mdcache/CMDCache.cpp

```diff
@@ -127,6 +130,8 @@ CMDCache::Reset()
 
 	Shutdown();
 	Init();
+
+	m_transientXmin = 0;
```

Comment:
Explain please why the "transient" and "Xmin" words are used or replace them with others

Reply:
TransactionXmin has nothing to do with the current transaction. it stores the number of the last committed transaction and in the case of read committed, it can change during the current transaction. in the case of repeatable read, TransactionXmin does not really change throughout the current transaction. therefore, it is correct to use xmin in the names, since this is exactly what we save. I replaced transient with temporary and transaction.